python/cudf/cudf/core/groupby/groupby.py

```diff
@@ -1359,7 +1359,7 @@ def _post_process_chunk_results(
         if isinstance(chunk_results, ColumnBase) or cudf.api.types.is_scalar(
             chunk_results[0]
         ):
-            data = {None: chunk_results}
+            data = {None: as_column(chunk_results)}
```

Comment:
Can't we `verify=False` here? It's a single column so the consistency checks are trivial and you're now turning this into a column.

Reply:
Good catch. Yeah added a `verify=False` here